CHANGELOG-developer.next.asciidoc

```diff
@@ -146,6 +146,7 @@ The list below covers the major changes between 7.0.0-rc2 and main only.
 - Improve compatibility and reduce flakyness of Python tests {pull}31588[31588]
 - Added `.python-version` file {pull}32323[32323]
 - Add support for multiple regions in GCP {pull}32964[32964]
+- Added LocationConstraint to functionbeat create bucket call {pull}33260[33260]
```

Comment:
I would remove this entry from developer changelog and copy paste it into the CHANGELOG.next.asciidoc file.